qcfractal/interface/models/common_models.py

```diff
@@ -6,7 +6,7 @@
 
 from pydantic import Schema, validator
 
-from qcelemental.models import Molecule, Provenance, ProtoModel
+from qcelemental.models import Molecule, Provenance, ProtoModel, AutodocBaseSettings
```

Comment:
Is `AutodocBaseSettings` used here?

Reply:
No, but we had talked about having all Elemental models pull from here at one point and importing them everywhere else in Fractal. So its not used here, but its imported by __init__ and then further imported elsewhere like `queue_manager`